src/Services/CRM/Currency/Result/CurrencyItemResult.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * This file is part of the bitrix24-php-sdk package.
+ *
+ * © Vadim Soluyanov <vadimsallee@gmail.com>
+ *
+ * For the full copyright and license information, please view the MIT-LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Bitrix24\SDK\Services\CRM\Currency\Result;
+
+use Bitrix24\SDK\Services\CRM\Common\Result\AbstractCrmItem;
+use Carbon\CarbonImmutable;
+
+/**
+ * Class CurrencyItemResult
+ *
+ * @property-read string $CURRENCY
```

Comment:
is this currency iso code? if this field currency iso-code, you can use Money\Currency